wirecard/qmore.py

```diff
@@ -74,9 +74,9 @@ def init_datastorage(self, orderIdent,
         return result
 
     def init_frontend(self, amount, currency, paymentType, language,
-                      orderDescription, successUrl, cancelUrl, failureUrl,
-                      serviceUrl, confirmUrl, consumerUserAgent,
-                      consumerIpAddress, autoDeposit=None,
+                      orderDescription, customerStatement, successUrl,
```

Comment:
I checked the usage and for invoices we use something else.
Are there other cases we use payment? I can only think of plans which leads to invoices and tickets